sass/styles.scss

```diff
@@ -57,6 +63,60 @@ div.middle {
     & a:hover {
         color: lightblue;
     }
+    & #download {
+        width: 100%;
+        box-sizing: border-box;
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: space-evenly;
+
+        & h2 {
+            color: var(--main-text1-color);
+            flex-basis: 100%;
+        }
+
+        & > div {
+            margin: 16px;
+            padding: 16px 0;
+            box-sizing: border-box;
+            flex-basis: 45%;
+            text-align: center;
+            background-color: #701e13;
+            border-radius: 10px;
+            border: 1px solid #ddb991;
+
+            &#download-steps {
```

Comment:
Why does the steps have to be treated specially?

Reply:
it's a special component to make it more obvious what to do next with download.